### PR TITLE
Fix SafeInfo network type

### PR DESCRIPTION
**Pull request by mikheevm**
This PR:
- Fixes inconsistency between the type and actual returned value for `network` prop in `getSafeInfo`

### CODE REVIEW

**Review by rmeissner — APPROVED** [review]

File: packages/safe-apps-sdk/src/types.ts [hunk 1]
```diff
@@ -59,11 +59,6 @@ export interface SdkInstance {
 }
 
 export interface SafeInfo {
-  safeAddress: string;
-  network: LowercaseNetworks;
```
Comment (germartinez) [inline packages/safe-apps-sdk/src/types.ts:63]: Should line 22 be removed?

File: packages/safe-apps-sdk/src/types.ts [hunk 2]
```diff
@@ -59,11 +59,6 @@ export interface SdkInstance {
...
-}
-
-export interface SafeInfoV1 {
   safeAddress: string;
   network: UppercaseNetworks;
```
Comment (rmeissner) [inline packages/safe-apps-sdk/src/types.ts:63]: Why is it not possible to have case insensitive networks?

File: packages/safe-apps-sdk/src/types.ts (hunk 1 above)
Comment (mikheevm) [reply packages/safe-apps-sdk/src/types.ts:63]: good catch

File: packages/safe-apps-sdk/src/types.ts (hunk 2 above)
Comment (mikheevm) [reply packages/safe-apps-sdk/src/types.ts:63]: Safe interface stores network names in uppercase and we want the type to be a list of constants (e.g "rinkeby", "mainnet", etc)

Comment (mikheevm) [reply packages/safe-apps-sdk/src/types.ts:63]: I think to have it truly case insensitive we need to switch to chain id :P

**Pull request merged**